terraform/main.tf

```diff
@@ -0,0 +1,63 @@
+data "azurerm_resource_group" "rg" {
+  name = var.resource_group_name
+}
+
+module "networking" {
+  source              = "./modules/networking"
+  vnet_name           = "my-vnet"
+  resource_group_name = data.azurerm_resource_group.rg.name
+  location            = "West Europe"
+  address_space       = ["10.0.0.0/16"]
+  subnets = [
+    {
+      name   = "exm-subnet-1"
+      prefix = "10.0.1.0/24"
+    },
+    {
+      name   = "rcb-subnet-1"
+      prefix = "10.0.2.0/24"
+    }
+  ]
+}
+
+module "security" {
+  source              = "./modules/security"
+  nsg_name            = var.nsg_name
```

Comment:
Daca ai ales sa pui valori in main.tf atunci trebuie facut pentru toate modulele la fel